docs/ospd-openvas.8

```diff
@@ -0,0 +1,141 @@
+.SH NAME
+ospd-openvas \- The OpenVAS Wrapper of the Greenbone Vulnerability Management
+.SH SYNOPSIS
+.BI "ospd-openvas [\|-v\|] [\|-h\|]  [\|-c " config-file\| "] [\|--log-file " log-file\| "]
+
+.SH DESCRIPTION
+.B Greenbone Vulnerability Management (GVM)
+is a vulnerability auditing and management framework made up of several modules.
+The OSPD OpenVAS Wrapper,
+.BR ospd-openvas
+is in charge of the communication between the scanner OpenVAS and the clients
+(GVMd and gvm-tools).
+
+
+.BR ospd-openvas
+inspects the remote hosts to list all the vulnerabilities and common
+misconfigurations that affects them.
+
+It is a command line tool with parameters to start a daemon which keep
+waiting for instructions to update the feed of vulnerability tests and
+ to start a scan.
+The second part of the interface is the redis store where the parameters
+about a scan task need to be placed and from where the results can be
+retrieved, being the unique communication channel between OSPD-OpenVAS
+and OpenVAS
+
+.SH OPTIONS
+.TP
+.BI "-s " <config-file> ", --config-file" <config-file>
+Use the alternate configuration file instead of
+.I ~/.config/ospd.conf
+
+.TP
+.B "--version"
+Prints the version number and exits
+
+.TP
+.B "-h, --help"
+Show a summary of the commands
+
+.TP
+.BI "-p " PORT ", --port "PORT
+TCP Port to listen on. Default: 1234
+
+.TP
+.BI "-b " ADDRESS ", --bind-address "ADDRESS
+Address to listen on. Default: 0.0.0.0
+
+.TP
+.BI "-u " UNIX_SOCKET ", --unix-socket "UNIX_SOCKET
+Unix file socket to listen on. Default: /tmp/ospd.sock
+
+.TP
+.BI "-m " SOCKET_MODE ", --socket-mode "SOCKET_MODE
+Unix file socket mode. Default: 0o700
+
+.TP
+.BI "-k " KEY_FILE ", --key-file "KEY_FILE
+Server key file. Default:
+/usr/var/lib/gvm/private/CA/serverkey.pem
+
+.TP
+.BI "-c " CERT_FILE ", --cert-file "CERT_FILE
+Server cert file. Default:
+/usr/var/lib/gvm/CA/servercert.pem
+
+.TP
+.BI "--ca-file "CA_FILE
+CA cert file. Default: /usr/var/lib/gvm/CA/cacert.pem
+
+.TP
+.BI "-L " LOG_LEVEL ", --log-level "LOG_LEVEL
+Wished level of logging. Default: WARNING
+
+.TP
+.BI "-f, --foreground"
+Run in foreground and logs all messages to console.
+
+.TP
+.BI "-l " LOG_FILE ", --log-file "LOG_FILE
+Path to the logging file.
+
+.TP
+.BI "--niceness "NICENESS
+Start the scan with the given niceness. Default 10
+
+.SH THE CONFIGURATION FILE
+
+The default
+.B ospd-openvas
+configuration file,
+.I ~/.config/ospd.conf
+contains these options under the secion [OSPD - openvas]:
+
+.IP log_level
+Wished level of logging.
+
+.IP socket_mode
+This option enables you to change the permissions on a socket.
+It must be set in octal format. E.g. socket_mode = 0o770
+
+.IP unix_socket
+This option is to specify the socket path.
+
+.IP log_file
+Path to the log file. If no log file is given, it will use the system log
+facility per default.
+
+.IP foreground
+If this option is set to yes, it logs in the standart output instead of logging
+in to a file or syslog.
```

Comment:
```suggestion
to a file or syslog.
```

I think it should be explained that the main purpose of `--foreground` is to control whether the daemon detaches and backgrounds itself, not to control where the log output goes.